package PBMessages;

message NextPrime
{
	required uint64 taskId = 1;
	required uint32 lastPrime = 2;
}
